webview/App.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { throttle } from './utils'
+
+declare var acquireVsCodeApi: any
```

Comment:
Should be `const`. Not real happy about the `any`, but there doesn't seem to be a good way around it.